guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -57,6 +77,29 @@ The `filename` option contains the full path to the file that downloads and exec
 |Grub2 UEFI HTTP | `http://_{smartproxy-example-com}_:8000/httpboot/grub2/grubx64.efi` | Requires the `httpboot` feature and renders the `filename` as a full URL where _{smartproxy-example-com}_ is a known host name of {SmartProxy} in {Project}.
 |=======
 
+=== Troubleshooting DHCP Problems in {Project}
+
+{Project} can manage an ISC DHCP server on internal or external DHCP {SmartProxy}. {Project} can list, create, and delete DHCP reservations and leases. However, there are a number of problems that you might encounter on occasions.
+
+.PXE loader option change
+
+When the PXE loader option is changed for an existing host, this causes a DHCP conflict. This is a known issue. Until this issue is fixed, the only workaround is to overwrite the DHCP entry.
```

Comment:
Can we link the known issue here?

Reply:
@lzap I've found the foreman issue for this, but there is no Bugzilla issue yet, so wrapping the foreman issue in an ifeval, but how will I handle the downstream info?